classes/Admin/Popups.php

```diff
@@ -1275,15 +1275,17 @@ public static function render_columns( $column_name, $post_id ) {
 					}
 					break;
 				case 'conversion_rate':
-					$opens       = $popup->get_event_count( 'open' );
-					$conversions = $popup->get_event_count( 'conversion' );
+					if ( ! pum_extension_enabled( 'popup-analytics' ) ) {
```

Comment:
@danieliser That column was already there and I just uncommented it and made some adjustments. 

Since we are already showing opens and conversions, it's only an extra mental calculation to get to the conversion rate so we aren't adding anything extra here other than making their life a bit easier by having it pre-calculated.

Are you advocating for not showing the conversion rate at all?

Reply:
Sorry to clarify I meant this line `if ( ! pum_extension_enabled( 'popup-analytics' ) ) {`.

Was asking if we should be doing that in core, or if we should be unhooking something from the analytics extension.

Works out the same either way, just a thought when reviewing.

Reply:
@danieliser The original open column using the same conditional as the current extension automatically adds data to those columns. If I remove that conditional in each of the three columns, then all of the data gets outputted twice, though with slightly different data.

So, to be compatible with the current version of the extension, we must keep that in the core plugin. 

That said, I do agree using a filter or hook here and then modifying the column in the extension does make more sense. Once the extension update is almost finished, we can explore so ways to handle being compatible with both the old version of the extension and the new version.